packages/addons/prettier/index.ts

```diff
@@ -24,7 +24,15 @@ export default defineAddon({
 		});
 
 		sv.file('.prettierrc', (content) => {
-			const { data, generateCode } = parseJson(content);
+			let data, generateCode;
+			try {
+				({ data, generateCode } = parseJson(content));
+			} catch {
+				log.warn(
+					'A `.prettierrc` config already exists and cannot be parsed. Skipping initialization.'
```

Comment:
People might be confused why it can't be parsed as valid yaml, so we should clarify we're attempting it as json
```suggestion
					'A `.prettierrc` config already exists and cannot be parsed as JSON. Skipping initialization.'
```